specification/appconfiguration/data-plane/readme.md

```diff
@@ -29,6 +29,24 @@ openapi-type: data-plane
 tag: package-1-0
 ```
 
+### Tag: package-2021-05-02
+
+These settings apply only when `--tag=package-featureManagement-1-0` is specified on the command line.
+
+``` yaml $(tag) == 'package-featureManagement-1-0'
```

Comment:
Considering there can be only a single tag when the generation is invoke I don't think we would be amble to generate a library that includes main REST API, feature management and keyVaultReference at the same time.

Reply:
@pakrym advice is welcome here. I separated these because they evolve separately and they are not direct resource types of our API and they also evolve separately. Do we need to be able to generate them at the same time for some reason?

Reply:
>Do we need to be able to generate them at the same time for some reason?

Yeah, data plane client libraries would need to generate all three at the same time.

The closes example I know (EventGrid) defines events in the same list as the main service swagger.

@johanste do you know of other services that document schemas that are not directly returned by service operations? (to give you some context, appconfig us trying to document configuration settings value conventions for feature flags and secret references)

Reply:
I went ahead and combined them in a new package tag to allow simultaneous generation.

Reply:
Sorry, this somehow got lost in my inbox. @pakrym, no off the top of my head, I'm not aware of any other azure services that document conventions in their swaggers. The fact that we are documenting it as part of the service makes it more than just a convention - this now becomes part of the service contract, and it would have to adhere to all of azure's versioning rules. Additionally, this is not using a discriminator/tagged union type to identify which "type" of object it is. I would expect an azure service to not only use the content-type to distinguish between different schemas.